Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -857,6 +973,53 @@ class AppStoreConnectService {
             .await()
     }
 
+    func pullTestFlightConfigurations() throws -> [TestFlightConfiguration] {
+        let apps = try listApps(bundleIds: [], names: [], skus: [], limit: nil)
+
+        return try apps.map {
+            let testers: [FileSystem.BetaTester] = try ListBetaTestersOperation(
+                options: .init(appIds: [$0.id])
+            )
+            .execute(with: requestor)
+            .await()
+            .map {
+                FileSystem.BetaTester(
+                    email: ($0.betaTester.attributes?.email)!,
+                    firstName: $0.betaTester.attributes?.firstName,
+                    lastName: $0.betaTester.attributes?.lastName
+                )
```

Comment:
You could simplify this line by creating a convenience `init` for `FireSystem.BetaTester` that takes the attributes struct as a param.